.github/workflows/build.yml

```
@@ -0,0 +1,35 @@
+---
+name: Build
+
+on:
+  push:
+    branches: ["master", "main"]
+  pull_request:
+    branches: ["master", "main"]
```

No need to limit this when it runs on a PR imo

```suggestion
  pull_request:
```